From f104c7b226d736c17f65227f2eb159a0cf186dcc Mon Sep 17 00:00:00 2001
From: "Angelo G. Del Regno" <kholk11@gmail.com>
Date: Sat, 11 Aug 2018 14:13:57 +0200
Subject: [PATCH 014/178] media: msm: camera_v2: Set regulator mode fast on
 poweron

To improve the regulation performance, set regulator mode to FAST
on poweron, to improve efficiency set to NORMAL on poweroff.

Change-Id: I776d2b8a022b3bf8c4966db0879cd9fa5cd59ef9
---
 drivers/media/platform/msm/camera_v2/common/cam_soc_api.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/msm/camera_v2/common/cam_soc_api.c b/drivers/media/platform/msm/camera_v2/common/cam_soc_api.c
index 0b3313237205..9b7ca886bb91 100644
--- a/drivers/media/platform/msm/camera_v2/common/cam_soc_api.c
+++ b/drivers/media/platform/msm/camera_v2/common/cam_soc_api.c
@@ -663,7 +663,7 @@ int msm_camera_regulator_set_mode(struct msm_cam_regulator *vdd_info,
 			CDBG("name : %s, enable : %d\n", tmp->name, mode);
 			if (mode) {
 				rc = regulator_set_mode(tmp->vdd,
-					REGULATOR_MODE_NORMAL);
+					REGULATOR_MODE_FAST);
 				if (rc < 0) {
 					pr_err("regulator enable failed %d\n",
 						i);
-- 
2.49.0

